packages/plugin-flex/src/__tests__/commands/flex/plugins/deploy.test.ts

```diff
@@ -164,8 +170,8 @@ describe('Commands/FlexPluginsDeploy', () => {
     expect(cmd.registerPluginVersion).toHaveBeenCalledTimes(1);
   });
 
-  it('should have flag as own property', () => {
-    expect(FlexPluginsDeploy.hasOwnProperty('flags')).toEqual(true);
+  it('should have own flags', () => {
+    expect(FlexPluginsDeploy.flags).not.toBeSameObject(FlexPlugin.flags);
```

Comment:
`not`? this seems to be the exact opposite of what this test is attempting to test for,

Reply:
This is checking that the command class has its own flags object which is not same as its parent class flags object (in reference, not value). Similar to what was being checked earlier with 'hasOwnProperty'. For some unknown reason that stopped working, so I have modified the test case implementation.

Reply:
What happens if `FlexPluginsDeploy.flags` is null? Wouldn't that come back as true?

Reply:
It could have been null previously too right? 😅 I just modified the implementation of the same test case. The test case was to see whether the class has its own property called 'flags'. We didn't check for non-null value there. I can update it, if we need that check

Reply:
A plugin can be `null` - this specific test for `deploy` for example, wouldn't. So can you confirm that too?